src/screens/MovieDetails/MovieDetails.js

```diff
@@ -32,6 +85,15 @@ export function MovieDetails({ route }) {
           <Button title={DOWNLOAD} />
         </TouchableHighlight>
         <Text style={styles.textOverview}>{route.params.item.overview}</Text>
+        <Text style={styles.titleRecommendMovies}>{RELATED}</Text>
+        <FlatList
+          horizontal={true}
+          onEndReached={loadMore}
+          data={data.pages.map((page) => page.data.results).flat()}
```

Comment:
Try extracting the data param so the code is more readable. Use a function to return the array of items you need here

Reply:
Done!